data/cube/View.js

```diff
@@ -46,6 +47,7 @@ export class View extends HoistBase {
     _rows = null;
     _rowCache = null;
     _leafMap = null; // Leaves, by source record id.
+    _simpleAggs = false;
```

Comment:
@lee based on our short conversation, I'm wondering does 'simple' mean the same thing as 'supports streaming updates'? If so, does it make sense to have that reflected in this field name>